src/components/cc-addon-features/cc-addon-features.js

```diff
@@ -91,8 +104,22 @@ export class CcAddonFeatures extends LitElement {
     return rawValue;
   };
 
-  // Here we sort feature by name (lower case) but first we force a specific order with SORT_FEATURES
+  /**
+   * @param {AddonFeaturesState} state
+   * @returns {state is (AddonFeaturesStateLoading|AddonFeaturesStateLoaded)}
+   * @private
+   */
+  _isLoadedOrLoading (state) {
+    return state.type === 'loaded' || state.type === 'loading';
+  }
```

Comment:
nitpick: since it's a rather simple method and to improve "reading code from just one place", I'd rather move this to a variable inside the render function.

Reply:
I will let @florian-sanders-cc correct me if I'm wrong, but you can't use a var because it will cause a TS error.